BasicTest/Program.cs

```diff
@@ -35,12 +35,12 @@ static void MassiveBulkOpsAsync(int AsyncOpsQty, bool preserveOrder, bool withCo
                 var conn = muxer.GetDatabase();
                 muxer.Wait(conn.PingAsync());
 
-#if DNXCORE50
+#if CORE_CLR
                 int number = 0;
 #endif
                 Action<Task> nonTrivial = delegate
                 {
-#if !DNXCORE50
+#if !CORE_CLR
```

Comment:
@agocke I don't have full context here, but the initial port noted it took far more time - has this changed? Link: https://github.com/StackExchange/StackExchange.Redis/commit/e10c20e2351900e3f8d7d88183bd479dd76b3c32